tests/cli/test_ivcfmerge_batch_cli.py

```diff
@@ -0,0 +1,34 @@
+import subprocess
+
+from hypothesis import given, strategies as st
+
+
+def test_merging_example_input(input_paths_file, ref_merged_path, tmp_path):
+    output_path = tmp_path / 'out.vcf'
+
+    subprocess.run(['python', 'ivcfmerge_batch.py', input_paths_file, str(output_path)])
+
+    with output_path.open() as outfile, open(ref_merged_path, 'r') as ref_merged:
+        assert outfile.read() == ref_merged.read()
+
+
+@given(batch_size=st.integers())
+def test_batch_sizes(input_paths_file, ref_merged_path, tmp_path, batch_size):
```

Comment:
@Zhicheng-Liu It would be best if we can test that the `--batch-size` arg is passed to `ivcfmerge_batch`, but I couldn't find a way to patch the `ivcfmerge_batch.py::ivcfmerge_batch` function.
Similarly for testing temp dir below

Reply:
The fact that the command is run in subprocess makes it hard/impossible to mock. I will accept this PR as it is.